src/modules/data/views/Transactions/TransactionList.tsx

```diff
@@ -52,13 +50,7 @@ export const TransactionList: React.FC<Props> = ({ filterString }) => {
   const [rowsPerPage, setRowsPerPage] = useState(PAGE_LIMITS[0]);
   const { createdFilter, lastEvent } = useContext(ApplicationContext);
 
-  const columnHeaders = [
-    t('hash'),
-    t('blockNumber'),
-    t('signer'),
-    t('status'),
-    t('dateMined'),
-  ];
+  const columnHeaders = [t('id'), t('type'), t('status'), t('dateMined')];
```

Comment:
"dateMined" is distinctly incorrect at this point ("Created" is probably more appropriate)